### DISCUSSION TITLE
Welcome to SwiftLink Discussions!

### DISCUSSION THREAD

**Discussion by mohammadKarimi** [mohammadKarimi/SwiftLink#75] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kamyab7** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Well, I'm just a 23yo .NET dev who is curious about system design and trying to create something that might be useful.

**Reply by mohammadKarimi** [reply]
Thank you @Kamyab7 . I am very happy to have you here.
